packages/shared/src/storage/appSetting.ts

```diff
@@ -0,0 +1,31 @@
+import { MMKV } from 'react-native-mmkv';
+
+import type { AsyncStorageStatic } from '@react-native-async-storage/async-storage';
+
+export const appSetting = new MMKV({
+  id: `onekey-app-setting`,
+});
```

Comment:
We need to consider how to migrate existing data to MMKV.
Otherwise, this will break existing installations once got released.

Reply:
No data migration is necessary. In app settings, mainly various scattered toggle values are stored.